src/platform/device.gni

```diff
@@ -124,6 +124,10 @@ declare_args() {
   chip_subscription_timeout_resumption = chip_persist_subscriptions
 }
 
+if (chip_enable_wifi == false) {
+  chip_device_config_enable_wifipaf = false
```

Comment:
you are resetting this later ... why not include this in the original condition like:

```
chip_device_config_enable_wifipaf = chip_enable_wifi && (current_os == "linux") 
```

Reply:
I will change it